.github/ISSUE_TEMPLATE/4-security-issue-disclosure.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
The security issue/disclosure template should make mention of the new process we have for handling security disclosures... Basically we don't want folks reporting security bugs here, immediately. We want them to instead send an email to the "SIR" team which will then take care of fixing, filing public issue, etc. I think ultimately the security template should exist to tell people to use that I don't think we should actually encourage folks to submit security issues through github yet.

Reply:
Yes, this exactly why I left it blank because I wasn't sure if we had a process. Can you slack me the info you'd like me to put on the template?